spark/src/main/scala/org/apache/comet/CometExecIterator.scala

```diff
@@ -95,7 +95,8 @@ class CometExecIterator(
     nativeUtil.getNextBatch(
       numOutputCols,
       (arrayAddrs, schemaAddrs) => {
-        nativeLib.executePlan(plan, arrayAddrs, schemaAddrs)
+        val ctx = TaskContext.get()
+        nativeLib.executePlan(ctx.stageId(), ctx.partitionId(), plan, arrayAddrs, schemaAddrs)
```

Comment:
is it to track times for specific partitions?

Reply:
Yes, because we create one plan per partition we have a lot of repeated plans and it is helpful to see the partition number.